### DISCUSSION TITLE
Window Store Version: Keep in Sync

### DISCUSSION THREAD

**Discussion by SR--** [mifi/lossless-cut#2245] [General]
@mifi I have paid for the Store version of your excellent app to support the project but at the moment it is 4 months behind GitHub. I shall have to switch back downloading releases from GitHub unless you commit to have it updated at the same time.

**Comment by mifi** [comment]
Thanks for supporting! The reason why it's a bit behind is because I like to iron out all bugs before releasing to the stores. 3.63.0 had a severe bug so I released a fix in 3.64.0 (3 weeks ago). I am planning to release this version to the stores soon, unless any other bugs come up. Is there any particular feature you're looking for that's currently missing in the store version?

See also https://github.com/mifi/lossless-cut/blob/master/issues.md#app-stores-and-github-difference

**Comment by SR--** [comment]
Nah, it is all good. Shall look forward to it then.

________________________________
